src/data-pipeline/utils/s3-asset.ts

```diff
@@ -22,7 +22,7 @@ export function uploadBuiltInSparkJarsAndFiles(
   bucket: IBucket,
   prefix: string,
 ) {
-  const jarFile = 'spark-etl-1.1.0.jar';
+  const jarFile = 'spark-etl-1.1.1.jar';
```

Comment:
How about using the version from `SolutionInfo.SOLUTION_VERSION_SHORT` or `SOLUTION_VERSION`?

Reply:
changed.